fidesctl/Dockerfile

```diff
@@ -1,57 +1,74 @@
-FROM --platform=linux/amd64 python:3.8-slim-buster
+FROM --platform=linux/amd64 python:3.8-slim-buster as base
 
-######################
-# Tool Installation ##
-######################
+# Update pip in the base image since we'll use it everywhere
+RUN pip install -U pip
+
+FROM base as builder
+#######################
+## Tool Installation ##
+#######################
 
-RUN apt-get update
-RUN apt-get install -y \
+RUN : \
+    && apt-get update \
+    && apt-get install \
+    -y --no-install-recommends \
     curl \
     git \
     ipython \
     make \
     vim \
     g++ \
     gnupg \
-    gcc
+    gcc \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-###########################
-## Database Dependencies ##
-###########################
+###################
+## Database Deps ##
```

Comment:
A huge nit, and you don't need to change it, but I strongly prefer _not_ to abbreviate things in stored procedures like `Dockerfile`s, scripts, etc. We should be using the long-form wherever possible, since 1) these things aren't being typed out by a human and 2) it improves readability.

Reply:
I agree, clarity > brevity